src/hearings/hearings.routes.ts

```diff
@@ -183,6 +184,14 @@ export const ROUTES: Routes = [
           title: 'HMCTS Manage cases | Amend Hearing | Check Answers'
         }
       },
+      {
+        path: 'hearing-confirmation',
+        component: HearingFinalConfirmationComponent,
+        canActivate: [HealthCheckGuard],
+        data: {
+          title: 'HMCTS Manage cases | Request Hearing | Confirmation'
+        }
+      }
```

Comment:
could we move this to a upper level if this can be used in other journey? i.e. 
hearings/hearing-confirmation